app/assets/javascripts/directives/filter_by_directive.js

```diff
@@ -0,0 +1,12 @@
+samson.directive('filterBy', function($timeout, $parse) {
```

Comment:
Yeah, I told myself to make sure I fix that before I pushed, but forgot. Fixed now.